extensions/sql-database-projects/src/models/options/deployOptionsModel.ts

```diff
@@ -7,251 +7,64 @@ import * as mssql from 'mssql';
-			, this.deploymentOptions.blockWhenDriftDetected.displayName
-			, this.deploymentOptions.blockOnPossibleDataLoss.displayName
-			, this.deploymentOptions.backupDatabaseBeforeChanges.displayName
-			, this.deploymentOptions.allowIncompatiblePlatform.displayName
-			, this.deploymentOptions.allowDropBlockingAssemblies.displayName
-			, this.deploymentOptions.dropConstraintsNotInSource.displayName
-			, this.deploymentOptions.dropDmlTriggersNotInSource.displayName
-			, this.deploymentOptions.dropExtendedPropertiesNotInSource.displayName
-			, this.deploymentOptions.dropIndexesNotInSource.displayName
-			, this.deploymentOptions.ignoreFileAndLogFilePath.displayName
-			, this.deploymentOptions.ignoreExtendedProperties.displayName
-			, this.deploymentOptions.ignoreDmlTriggerState.displayName
-			, this.deploymentOptions.ignoreDmlTriggerOrder.displayName
-			, this.deploymentOptions.ignoreDefaultSchema.displayName
-			, this.deploymentOptions.ignoreDdlTriggerState.displayName
-			, this.deploymentOptions.ignoreDdlTriggerOrder.displayName
-			, this.deploymentOptions.ignoreCryptographicProviderFilePath.displayName
-			, this.deploymentOptions.verifyDeployment.displayName
-			, this.deploymentOptions.ignoreComments.displayName
-			, this.deploymentOptions.ignoreColumnCollation.displayName
-			, this.deploymentOptions.ignoreAuthorizer.displayName
-			, this.deploymentOptions.ignoreAnsiNulls.displayName
-			, this.deploymentOptions.generateSmartDefaults.displayName
-			, this.deploymentOptions.dropStatisticsNotInSource.displayName
-			, this.deploymentOptions.dropRoleMembersNotInSource.displayName
-			, this.deploymentOptions.dropPermissionsNotInSource.displayName
-			, this.deploymentOptions.dropObjectsNotInSource.displayName
-			, this.deploymentOptions.ignoreColumnOrder.displayName
-			, this.deploymentOptions.ignoreTablePartitionOptions.displayName
-			, this.deploymentOptions.doNotEvaluateSqlCmdVariables.displayName
-			, this.deploymentOptions.disableParallelismForEnablingIndexes.displayName
-			, this.deploymentOptions.disableIndexesForDataPhase.displayName
-			, this.deploymentOptions.restoreSequenceCurrentValue.displayName
-			, this.deploymentOptions.rebuildIndexesOfflineForDataPhase.displayName
-			, this.deploymentOptions.preserveIdentityLastValues.displayName
-			, this.deploymentOptions.isAlwaysEncryptedParameterizationEnabled.displayName
-			, this.deploymentOptions.allowExternalLibraryPaths.displayName
-			, this.deploymentOptions.allowExternalLanguagePaths.displayName
-			, this.deploymentOptions.hashObjectNamesInLogs.displayName
-			, this.deploymentOptions.doNotDropWorkloadClassifiers.displayName
-			, this.deploymentOptions.ignoreWorkloadClassifiers.displayName
-			, this.deploymentOptions.ignoreDatabaseWorkloadGroups.displayName
-			, this.deploymentOptions.doNotDropDatabaseWorkloadGroups.displayName].sort();
+	constructor(private defaultOptions: mssql.DeploymentOptions) {
```

Comment:
I didn't see defaultOptions used elsewhere in this PR? Does defaultOptions get used anywhere else? If not, then a private variable doesn't need to be created for it here

Reply:
This var is just used to get a immutable copy of options to the local var only!